doc/developer/release.rst

```diff
@@ -1,26 +1,26 @@
 Release Process
 ===============
 
-- Update the release notes:
+- Set release variables:
 
-  1. Review and cleanup ``doc/release/release_dev.rst``,
+      export VERSION=<version number>
+      export PREVIOUS=<previous version number>
+      export ORG="networkx"
+      export REPO="networkx"
 
-  2. Fix code in documentation by running
-     ``cd doc && make doctest``.
+- Autogenerate release notes:
 
-  3. Make a list of merges and contributors by running
-     ``doc/release/contribs.py <tag of previous release>``.
+      changelist ${ORG}/${REPO} networkx-${PREVIOUS} main --version ${VERSION}  --out doc/release/release_dev.rst --format rst
 
-  4. Paste this list at the end of the ``release_dev.rst``. Scan the PR titles
-     for highlights, deprecations, and API changes, and mention these in the
-     relevant sections of the notes.
+- Update the release notes for final release (skip this step
+  if you are making a pre-release):
 
-  5. Rename to ``doc/release/release_<major>.<minor>.rst``.
+  1. Rename to ``doc/release/release_<major>.<minor>.rst``.
 
-  6. Copy ``doc/release/release_template.rst`` to
+  2. Copy ``doc/release/release_template.rst`` to
```

Comment:
Do we need to keep a `release_dev.rst` file around anymore?  It gets overwritten by the `changelist` command above, right?

Reply:
See 93fa48486055d2361d8b1844a468cecdf655dc35